packages/create/src/helpers.ts

```diff
@@ -383,6 +335,128 @@ async function checkPostgresDbExists(options: any, root: string): Promise<true>
     return true;
 }
 
+/**
+ * Check to see if Docker is installed and running.
+ * If not, attempt to start it.
+ * If that is not possible, return false.
+ *
+ * Refs:
+ * - https://stackoverflow.com/a/48843074/772859
+ */
+export async function isDockerAvailable(): Promise<{ result: 'not-found' | 'not-running' | 'running' }> {
+    const dockerSpinner = spinner();
+
+    function isDaemonRunning(): boolean {
+        try {
+            execFileSync('docker', ['stats', '--no-stream'], { stdio: 'ignore' });
+            return true;
+        } catch (e: any) {
+            return false;
+        }
+    }
+
+    dockerSpinner.start('Checking for Docker');
+    try {
+        execFileSync('docker', ['-v'], { stdio: 'ignore' });
+        dockerSpinner.message('Docker was found!');
+    } catch (e: any) {
+        dockerSpinner.stop('Docker was not found on this machine. We will use SQLite for the database.');
+        return { result: 'not-found' };
+    }
+    // Now we need to check if the docker daemon is running
+    const isRunning = isDaemonRunning();
+    if (isRunning) {
+        dockerSpinner.stop('Docker is running');
+        return { result: 'running' };
+    }
+    dockerSpinner.message('Docker daemon is not running. Attempting to start');
+    // detect the current OS
+    const currentPlatform = platform();
+    try {
+        if (currentPlatform === 'win32') {
+            // https://stackoverflow.com/a/44182489/772859
+            execSync('"C:\\Program Files\\Docker\\Docker\\Docker Desktop.exe"');
+        } else if (currentPlatform === 'darwin') {
+            execSync('open -a Docker');
+        } else {
+            execSync('systemctl start docker');
+        }
```

Comment:
Might be worth to `enable` instead of `start` to make docker auto-start on next reboot. Sidenote: Both starting and enabling require root privileges which might trip up some OS'

Reply:
This command is running only on Linux based OS. In most cases the primary user has administrator rights. So again, covering 100% of setups (especially Linux-based) is virtually impossible. I think we can iterate here based on user feedback